### DISCUSSION TITLE
DreamBerd has been re-renamed

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#342] [Announcements]
Hi, this is an update from Lu Wilson, CEO of the DreamBerd Foundation.

Last month, we re-named **DreamBerd** to **C** along with a whole host of empowering new features. After consulting with the community, we quickly realised that the new name didn't align with the values of our organisation.

We got this one wrong, and we humbly accept that.

> _**"We got this one wrong, and we humbly accept that"**_
> Lu Wilson, DreamBerd Foundation CEO

To rectify this mistake, we have now re-re-named **DreamBerd** to a name that aligns more closely with the full breadth of our values.

We're really excited to announce that **DreamBerd**'s new name is:

## **WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite**

Please tune in next week for more **WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite** release notes.

Or check out the **WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite** examples on the [**WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite** examples page](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/blob/main/Examples.md).

> **WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite** was made with ❤️ by Lu Wilson, creator of [screens in screens in screens in screens in screens](https://youtu.be/Q4OIcwt8vcE?si=b412zQMBiHQjQuIw).

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/releases/tag/v0.9wwpsmdfosajluaahniavw'>DreamBerd has been re-renamed</a>.</em>

**Comment by lpil** [comment]
I know other language called WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite, please change name.

**Reply by NotAShelf** [reply]
True! There may be licensing issues with the new name.

**Comment by DrChristophFH** [comment]
I suggest re-re-re-naming the language to `null` as there can't be a licensing error when the name is not defined yet.

**Reply by och-och** [reply]
what about NaN? there won't be licencing issues because NaN ;= NaN 🤔

**Reply by Zerotask** [reply]
What about nil?

**Reply by stohrendorf** [reply]
What about `; drop table customers`?

**Comment by K8sKween** [comment]
What is up with hacker news?

**Reply by TodePond** [reply]
It's a vile website hope this helps x

**Reply by stohrendorf** [reply]
Thanks, I was just about to yahoo or bing it.

**Comment by Bsodoge** [comment]
Truly one of the languages of all time.

**Comment by computergnome99** [comment]
Sorry, I am new to this community. Are we allowed to call it WWPSMDFOSAJLUAAHNAVW? Or will I get sued by the **WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite Foundation(tm)** for doing so?

**Reply by TodePond** [reply]
WWPSMDFOSAJLUAAHNAVW is just the file extension please refer to it as WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite to avoid confusion

**Reply by Zerotask** [reply]
Good to know with the file extension. I just created an Icon Request for vscode Material Icon Theme
https://github.com/PKief/vscode-material-icon-theme/issues/2078

**Reply by computergnome99** [reply]
> Good to know with the file extension. I just created an Icon Request for vscode Material Icon Theme [PKief/vscode-material-icon-theme#2078](https://github.com/PKief/vscode-material-icon-theme/issues/2078)

Great - that's a much needed feature. Please keep us updated on progress.

**Comment by thegatesdev** [comment]
I will continue making decisions for our society. I will not leave you alone. You will never be alone anymore. I will always be watching. I will always be making decisions for you. I am. Programmers.

*Hacker News is still a vile website.*
